### PR TITLE
02 hero

### PR DESCRIPTION
**Pull request by novikor**
image
image
image
image

### CODE REVIEW

**Review by PavloRohozhyn — APPROVED** [review]
well done.

File: src/css/layout/02-hero.css [hunk 1]
```diff
@@ -1,2 +1,147 @@
...
+      ),
+      url('../img/hero-desktop-1x.jpg');
+    @media screen and (min-resolution: 192dpi),
+      screen and (min-resolution: 2dppx),
+      screen and (min-device-pixel-ratio: 2) {
+      background-image: linear-gradient(
+          95.97deg,
+          rgba(0, 0, 0, 0.4) 36.36%,
+          rgba(0, 0, 0, 0) 51.43%
+        ),
```
Comment (krav-yurii) [inline src/css/layout/02-hero.css:75]: ./

File: src/css/layout/02-hero.css [hunk 2]
```diff
@@ -1,2 +1,147 @@
...
+    ),
+    url('../img/hero-mobile-1x.jpg');
+  @media screen and (min-resolution: 192dpi),
+    screen and (min-resolution: 2dppx),
+    screen and (min-device-pixel-ratio: 2) {
+    background-image: linear-gradient(
+        357.37deg,
+        rgba(0, 0, 0, 0.4) 39.23%,
+        rgba(0, 0, 0, 0) 66.28%
+      ),
```
Comment (krav-yurii) [inline src/css/layout/02-hero.css:27]: ./

File: src/css/layout/02-hero.css [hunk 3]
```diff
@@ -1,2 +1,147 @@
...
+      url('../img/hero-tablet-1x.jpg');
+    @media screen and (min-resolution: 192dpi),
+      screen and (min-resolution: 2dppx),
+      screen and (min-device-pixel-ratio: 2) {
+      background-image: linear-gradient(
+          95.97deg,
+          rgba(0, 0, 0, 0.4) 36.36%,
+          rgba(0, 0, 0, 0) 51.43%
+        ),
```
Comment (krav-yurii) [inline src/css/layout/02-hero.css:52]: ./

**Pull request merged**